wiki/pl.json

```diff
@@ -3,9 +3,9 @@
 	"achievement-page-description": "Lista osiągnięć Manufactur'inc na Steam.",
 	"contracts-page-title": "Kontrakty",
 	"contract-page-title": "Kontrakt",
-	"equipments-page-title": "Equipments", //Review and context needed. What do you mean by equipment? Machines? 
-	"equipment-page-title": "Equipment", //^^
-	"equipment-page-description": "Ta karta pokazuje informację na temat ekwipunku \"%element%\" w Manufactur'inc.", //Review and context needed
+	"equipments-page-title": "Wyposażenie", 
```

Comment:
I see you put the same word on both, the first is plural and the second is singular. If you language makes the difference here 😅

Reply:
Thanks for the confirmation :)